webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/controllers/VolumeTracingController.scala

```diff
@@ -144,15 +149,24 @@ class VolumeTracingController @Inject()(
                  tracingId: String,
                  volumeDataZipFormat: String,
                  version: Option[Long],
-                 voxelSize: Option[String]): Action[AnyContent] =
+                 voxelSize: Option[String],
```

Comment:
So this should be called `voxelSizeFactor`, or am I wrong here?
Because in line 160 it is converted to a `voxelSIzeFactorParsedOpt`